face/VatomView.js

```diff
@@ -59,10 +85,11 @@ export default class VatomView {
   }
 
   load() {
+    
```

Comment:
To prevent the case of multiple error or loader views being added somehow, can you add something like this to the start of this function?

``` js
// Reset loader and error views
if (this.loader && this.loader.parentNode) this.loader.parentNode.removeChild(this.loader)
if (this.errorView && this.errorView.parentNode) this.errorView.parentNode.removeChild(this.errorView)
this.loader = null
this.errorView = null
```